testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -2376,6 +2370,9 @@ def update_props():
         else:
             log_info("Non conflict revision exist {} ".format(revs))
             assert False
+    for doc in sdk_update_docs2:
```

Comment:
this seems duplicate operation as line 2377? not sure if you need to do it twice, but I see this block looks the same as block starting line 2377 except an assert been added